Lib/idlelib/mainmenu.py

```diff
@@ -94,11 +94,12 @@
 
  ('options', [
    ('Configure _IDLE', '<<open-config-dialog>>'),
-   ('_Code Context', '<<toggle-code-context>>'),
+   None,
+   ('Zoom Height', '<<zoom-height>>'),
```

Comment:
I suggested reversing order, but could be persuaded otherwise. Zoom Height is also a toggle and we could toggle its label to Un-zoom Height?, unZoom Height? Normal Height?  Revert Height?   What do you think.  If so, issue would be 'move Zoom Height to Options with Code Context and make both labels switch.  Note: if we add colorizer toggle (as has been requested) we should put it here also, with, Colorizer on,off or  Start/Stop Colorizer.

Reply:
I don't have a good argument for not reversing them, so with the latest commit, I switched them.  :-)  I also went with `Restore Height` on the toggle for `Zoom Height`.  It seemed to make sense since 'Zoom' was a verb.  The changing of the label for `Zoom Height` was a little more complex.  I tested the Alt-2 on the browser tree and it didn't have a label, so I think it's good there.

I also found a way to index the menu items without using a number.  I'm quite pleased with this result because I didn't like using the number.  Plus, I realized using a number on the Options menu would have broken MacOS.  On the [man page](https://www.tcl.tk/man/tcl8.6/TkCmd/menu.htm#M23) for `menu` (the link should take you to the 'Widget Command' section), it describes the different valid values for index, including pattern.  Bingo!

Reply:
'Restore'  is fine, better than any of my suggestions.

I had forgotten about Mac manipulations, but I also did not like the hard-coded indexes.  The Shipman NMT 8.5 reference does not specify the allowed menu item 'indexes'.